syntaxes/nim.json

```diff
@@ -1473,12 +1476,16 @@
       },
       "name": "string.quoted.triple.raw.nim"
     },
-    "float_literal": {
+    "custom_literal": {
       "patterns": [
         {
-          "match": "\\b\\d[_\\d]*((\\.\\d[_\\d]*([eE][\\+\\-]?\\d[_\\d]*)?)|([eE][\\+\\-]?\\d[_\\d]*))('([fF](32|64|128)|[fFdD]))?",
```

Comment:
Does the exponentiation part need to be repeated when it seems like the hexadecimal/octal/etc is an optional suffix in its own right?